app/Http/Resources/FamiliaProfesionalResource.php

```diff
@@ -14,6 +14,8 @@ class FamiliaProfesionalResource extends JsonResource
      */
     public function toArray(Request $request): array
     {
-        return parent::toArray($request);
+        return array_merge(parent::toArray($request), [
+            'ciclos' => new CicloResource($this->cicloResource),
```

Comment:
Sería `$this->ciclos` en lugar de `$this->cicloResource`.

Es posible que lo modificaras porque te generara algún error, si fuera así, pregúntame por qué mañana.